endless/endless.h

```diff
@@ -26,7 +26,7 @@ EOS_SDK_DEPRECATED_IN_0_0
 gboolean eos_hello_sample_function (GFile   *file,
                                     GError **error);
 
-EOS_SDK_AVAILABLE_IN_0_6
+EOS_SDK_DEPRECATED_IN_0_6
```

Comment:
Should we consider removing this function entirely, or just making it `return FALSE`?

Reply:
I'd rather keep it to avoid mishaps for now, but that's a good thought about returning FALSE. I added that and squashed it into the previous commit.